packages/xo-server/src/api/xosan.js

```diff
@@ -6,7 +6,7 @@ import execa from 'execa'
 import fs from 'fs-extra'
 import map from 'lodash/map'
 import { tap, delay } from 'promise-toolbox'
-import { NULL_REF } from 'xen-api'
+import { NULL_REF } from '@xen-orchestra/xapi'
```

Comment:
Not sure this change is necessary, `xen-api` is a peer dependency of `@xen-orchestra/xapi`, which means it's expected to be a dependency of `xo-server`.

Reply:
Do you want us to use the two libraries in the same time?

Reply:
That's a possibility, as I said, I'm not sure about it.

Let's avoid introducing unnecessary changes in `@xen-orchestra/xapi` for now, we'll be able to do it later if we change our mind :slightly_smiling_face:

Reply:
Why I shouldn't do it? Please give me the reason for not only using `@xen-orchestra/xapi`.

Reply:
Because you can import it directly from `xen-api` instead.

Reply:
But `@xen-orchestra/xapi` is a hight level layer of the `xen-api`. So, we don't need to use the two libraries for that.

Reply:
Yes, but:
1. `xen-api` is a peer dependency which means it's still available to the consumer code
2. it introduces constraints between `@xen-orchestra/xapi` and `xen-api`, which means that each changes in the low-level one requires a change (and a new release) of the high-level one.

IMHO, it's best to use `xen-api` directlty when possible.